ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/registry-nutanix-object/cucushift-installer-rehearse-nutanix-ipi-registry-nutanix-object-workflow.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/nutanix/ipi/registry-nutanix-object/cucushift-installer-rehearse-nutanix-ipi-registry-nutanix-object-workflow.yaml",
```

Comment:
Suggest to rename `registry-nutanix-object` folder, how about `image-registry-s3`?